armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -472,6 +473,12 @@ def _checkBlockSimilarity(self):
             return True
 
 
+def getBlockNuclideTemperature(block, nuclide):
+    """Return the average temperature for 1 nuclide."""
+    tempIntegratedVolume, volume = getBlockNuclideTemperatureAvgTerms(block, [nuclide])
+    return tempIntegratedVolume / volume
```

Comment:
If a block doesn't have the given nuclide, `volume` will come back as 0.0.
```suggestion
    return tempIntegratedVolume / volume if volume >0.0 else 0.0
```

Reply:
addressed